mmcv/runner/hooks/logger/text.py

```diff
@@ -36,6 +41,13 @@ def _get_max_memory(self, runner):
         return mem_mb.item()
 
     def _log_info(self, log_dict, runner):
+        if runner.meta is not None and 'exp_name' in runner.meta:
+            if (self.every_n_inner_iters(
+                    runner,
+                    self.interval_exp_name)) or self.end_of_epoch(runner):
+                exp_info = 'Exp name: {}\t'.format(runner.meta['exp_name'])
```

Comment:
Use f-string instead format.

Reply:
I guess python3.5 may not support f-string?